atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/api/TransactionManager.java

```diff
@@ -379,12 +394,14 @@ default boolean isInitialized() {
      *
      * @throws IllegalStateException if the transaction manager has been closed.
      */
+    // I would say we don't really care about this? If we want this metric it should be in the puncher store.
```

Comment:
I think there was an issue a while back where this actually turned out to be helpful (technically this could end up doing arbitrarily large range scans on the `_punch` table), so we may want to keep this in some form - though the internal database dashboard should be enough to identify this based on range scan metrics. @gmaretic what do you think?

Reply:
My point was that this just delegates to the puncher, so probably should have it there.

Reply:
re: whitelist vs blacklist: if someone doesn't think about it from the start, probably we don't need this metric. I will do a pass over the other ones and eventually deprecate the one without filtering, so you will always have @Timed on some method in the interface, I think that's enough of a ping.